plugin/pkg/shoot/validator/admission.go

```diff
@@ -497,13 +521,13 @@ func (c *validationContext) validateScheduling(ctx context.Context, a admission.
 	return nil
 }
 
-func getNumberOfShootsOnSeed(shootLister gardencorelisters.ShootLister, seedName string) (int64, error) {
+func getNumberOfShootsOnSeed(shootLister gardencorev1beta1listers.ShootLister, seedName string) (int64, error) {
 	allShoots, err := shootLister.Shoots(metav1.NamespaceAll).List(labels.Everything())
 	if err != nil {
 		return 0, fmt.Errorf("could not list all shoots: %w", err)
 	}
 
-	seedUsage := helper.CalculateSeedUsage(allShoots)
+	seedUsage := v1beta1helper.CalculateSeedUsage(allShoots)
```

Comment:
Shouldn't this be the internal version?

Reply:
`shootLister` returns the versioned shoot not the `core` shoot. It can be done but then we will have to convert the shootList to core shoot list.